wys/atr/_sql/create-table-wys_atr_kertcher.sql

```diff
@@ -0,0 +1,19 @@
+/*
+Creates table to store data from xlsx files in
+L:\TDCSB\PROJECT\DATA COLLECTION GROUP\3 - MANUAL COUNTS\2020 Manual Counts\WYS SIGN OBSERVANCE\Kertcher
+
+Files were read in and coverted to csv in notebook:
+wys/atr/notebooks/tmc_data_preprocessing.ipynb
+*/
+
+CREATE TABLE data_requests.wys_atr_kertcher
```

Comment:
Why is this in `data_requests`? It should be in `data_analysis`.

In general I be wary of labelling objects with other employees' names. It implies an ownership that doesn't necessarily exist and which I don't find to be particularly helpful. In this case all she did was validate the manual counts, there's nothing particularly useful in labelling the table with her name. I would suggest `wys_validation_counts` instead.

Reply:
oh! i didn't know that was her name oops! I thought it was related to the study.